Sources/CollectionViewManager.swift

```diff
@@ -578,7 +586,7 @@ open class CollectionViewManager: NSObject {
                      completion: Completion? = nil) {
         func insert(in collectionView: UICollectionView?) {
             for (sectionItem, index) in zip(sectionItems, indexes) {
-                sectionItemsProvider.insert(sectionItem, at: index)
+                sectionItemsWrapper.insertSectionItem(sectionItem, at: index)
```

Comment:
I would prefer omit section item from function name because type is strictly provided
```suggestion
                sectionItemsWrapper.insert(sectionItem, at: index)
```

Reply:
renamed